pyaedt/edb_core/components.py

```diff
@@ -803,14 +803,9 @@ def create_port_on_pins(self, refdes, pins, reference_pins, impedance=50.0):
 
     @pyaedt_function_handler()
     def create_port_on_component(
-        self,
-        component,
-        net_list,
-        port_type=SourceType.CoaxPort,
-        do_pingroup=True,
-        reference_net="gnd",
+        self, component, net_list, port_type=SourceType.CoaxPort, do_pingroup=True, reference_net="gnd", port_name=None
     ):
-        """Create ports on given component.
+        """Create ports on a given component.
```

Comment:
```suggestion
        """Create ports on a component.
```
We avoid using "on a given" or "on a specific" component to reduce wordiness. The ``component`` parameter makes it clear that you specify an EDB component.